rakefile.rb

```diff
@@ -38,32 +38,42 @@
 approval_tests = "tests/FakeItEasy.Tests.Approval/bin/Release/FakeItEasy.Tests.Approval.dll"
 
 repo = 'FakeItEasy/FakeItEasy'
+ci_server_url = 'http://teamcity.codebetter.com/admin/editBuildParams.html?id=buildType:bt929'
 release_issue_labels = ['P2', 'build', 'documentation']
-release_issue_body = <<-eos
-**Ready** when all other issues on this milestone are **Done** and closed.
+release_issue_common_steps = <<-eos
+**in-progress** when all other issues on this milestone are closed.

```

Comment:
For some reason I thought we'd replaced **ready** with **in-progress**, when really **ready** survives and I think **working** or something had been renamed. I've been out to lunch this whole time. I would've left the original had I been on the ball. :blush:

I much prefer your new version, though.